react-frontend/src/login/PreLoginPanelComponent.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { Col, Row, Container } from "core-components/grid/GridComponent";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+import PeerlyTextAndLogo from "shared-components/peerly-logo/PeerlyTextAndLogo";
+import InformativeTextComponent from "login/InformativeTextComponent";
+import LoginImageComponent from "login/LoginImageComponent";
+
+const Wrapper = styled(Container)`
+  height: 100vh;
+  background-color: ${({ theme }) =>
+    theme === "dark" ? "var(--atomic)" : "var(--white)"};
+`;
+
+const RowComponent = styled(Row)`
```

Comment:
@avi4630 - when you are splitting the row into 3 equal sections. Don't you think it's a requirement for using `flex-column` with `content-around` ???

Reply:
@sahilbhatia   when i used flex-column then the component not render at center therefore d-flex 
is required